components/automate-ui/src/app/pages/api-token/details/api-token-details.component.scss

```diff
@@ -40,11 +40,30 @@ chef-tab-selector {
 
 chef-form-field {
   width: 240px;
+
+  padding-bottom: 30px;
+}
+
+// nest? it's only used in chef-form-field
```

Comment:
Does not matter, as we have no particular convention one way or the other.

Reply:
I like nesting where I can since it just adds a little more context to what the CSS rule is for